opentreemap/treemap/views/misc.py

```diff
@@ -117,6 +118,18 @@ def boundary_to_geojson(request, instance, boundary_id):
     return HttpResponse(geom.geojson)
 
 
+def add_anonymous_boundary(request):
+    request_dict = json.loads(request.body)
+    polygon = Polygon(request_dict.get('polygon', []))
+    b = Boundary()
+    b.geom = MultiPolygon(polygon)
+    b.name = ''
+    b.category = ''
+    b.sort_order = 1
+    b.save()
```

Comment:
I also think that using a custom manager to encapsulate the logic for named vs. anonymous boundaries is a good idea.

Note that since Django 1.7 you can create a custom manager from a `QuerySet`, which allows the custom methods to be called before or after other ORM methods, instead of requiring that they be first:
```python
class BoundaryQuerySet(models.QuerySet):
    def create_anonymous_boundary(self, coordinates):
        polygon = Polygon(request_dict.get('polygon', []))
        return self.create(geom=MultiPolygon(polygon),
                           name='',
                           category='',
                           sort_order = 1,
                           searchable=False)

    def anonymous(self):
        return self.filter(name='', category='')

    def named(self):
        return self.filter(name='', category='')

...

class Boundary(models.Model):
    ...
    manager = GeoManager.from_queryset(BoundaryQuerySet)()